web_widget_timepicker/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================
+Timepicker widget in form view
+==============================
+
+This module defines a timepicker widget, to be used with float fields
+or (function) fields. Use ``widget='timepicker'`` in your form view
+definition. It can be use as a replacement for the standard float_time widget.
+
+If you use the widget with a field record, the input field has the following default
+timepicker options:
+
+* By default the possible selection is based on 15 minute interval (step: 15)
+* By default 24 hour mode with H:i format (timeFormat: 'H:i')
+* By default scroll selection starts at current time (scrollDefault: 'now')
+
+The widget uses the jquery.timepicker plugin by Jon Thornton
+
+
+Usage
+=====
+
+This module defines a new widget type for form views input fileds.
+Set the attribute ``widget=timepicker`` in a ``field`` tag in a form view.
+
+You can pass custom options through the "timepicker" field in the options attribute:
+
+    ...
```

Comment:
remove the colons and double the colon after `attribute` above: http://docutils.sourceforge.net/docs/user/rst/quickstart.html#preformatting-code-samples